provider/lxd/provider.go

```diff
@@ -177,11 +177,6 @@ func (p *environProvider) Ping(ctx context.ProviderCallContext, endpoint string)
 		return errors.Trace(err)
 	}
 
-	// Make sure we have an https url
-	if lxdEndpoint != endpoint {
-		return errors.Errorf("invalid URL %q: only HTTPS is supported", endpoint)
```

Comment:
can we parse url here to check it has a https scheme or is that already handled by `lxd.EnsureHostPort`?

Reply:
Yep, already done by `lxd.EnsureHostPort`.